modules/javafx.media/src/main/native/gstreamer/gstreamer-lite/gstreamer/gst/gst_private.h

```diff
@@ -532,7 +532,11 @@ struct _GstDynamicTypeFactoryClass {
 struct _GstClockEntryImpl
 {
   GstClockEntry entry;
+#if defined (GSTREAMER_LITE) && defined(LINUX)
```

Comment:
Why is this qualified with `defined(LINUX)`? Unless the usage of this variable is different based on the platform, I would expect that the type is wrong on all platforms. Am I missing something?

Reply:
This is old code which was restored from GStreamer 1.20.1 and only needed on Linux to support older GLib.
See https://github.com/openjdk/jfx/pull/1290.

Reply:
Thanks for the explaination.